misk/src/main/kotlin/misk/cloud/fake/environment/FakeInstanceMetadataModule.kt

```diff
@@ -1,9 +1,10 @@
-package misk.environment
+package misk.cloud.fake.environment
 
 import com.google.inject.AbstractModule
+import misk.environment.InstanceMetadata
 
 /** Provides a hard-coded set of instance metadata */
-class StaticInstanceMetadataModule(val metadata: InstanceMetadata) : AbstractModule() {
+class FakeInstanceMetadataModule(val metadata: InstanceMetadata) : AbstractModule() {
```

Comment:
Philosophical point - do we want fakes in `/src/main` versus `/src/test`, and made available as a test artifact?